glambie/processing/process_regional_results.py

```diff
@@ -207,6 +212,8 @@ def _run_region_timeseries_one_source(data_catalogue_annual: DataCatalogue,
         Glambie Data Group which is calculated
     output_path_handler : OutputPathHandler
         object to handle output path. If set to None, no plots / other data will be saved
+    min_max_time_window_for_longterm_trends : Tuple[float, float], optional
+        if specified, longterm trends are not exceeding these dates
```

Comment:
I.e. longterm trends are filtered to the min/max dates. Or an error is thrown if the longterm trends don't cover enough of a time span? What format are the dates expected in (e.g. why not datetime).

Reply:
How does the longterm trend window impact the timeseries? Are all results limited to this window? Are no longterm corrections applied outside of the window?

Reply:
i clarified in the docstring